src/test/java/core/basesyntax/HelloWorldTest1.java

```diff
@@ -0,0 +1,183 @@
+                registrationService.register(user));
+        String excepted = "User login is need to be at least 6 letters!";
+        String actual = exception.getMessage();
+        assertTrue(actual.contains(excepted));
+    }
+
+    @Test
+    void fiveLettersLogin_NotOk() {
+        User user = new User();
+        user.setLogin("abcdf");
+        user.setPassword("messagetotheworld");
+        user.setAge(41);
+        Exception exception = assertThrows(CustomeException.class, () ->
+                registrationService.register(user));
+        String excepted = "User login is need to be at least 6 letters!";
+        String actual = exception.getMessage();
+        assertTrue(actual.contains(excepted));
+    }
+
+    @Test
+    void eightLettersLogin_Ok() {
+        User user = new User();
+        user.setLogin("Vitaliks");
+        user.setAge(22);
+        user.setPassword("normalpassword321");
+        registrationService.register(user);
+        assertTrue("Vitaliks".equals(user.getLogin()));
+    }
+
+    @Test
+    void emptyloginUser_NotOk() {
+        User user = new User();
+        user.setLogin("");
+        user.setAge(32);
+        user.setPassword("thebestpassword");
+        Exception exception = assertThrows(CustomeException.class, () ->
+                registrationService.register(user));
+        String excepted = "User login or password is empty";
+        String actual = exception.getMessage();
+        assertTrue(actual.contains(excepted));
+    }
+
+    @Test
+    void userOk_Ok() {
+        User user = new User();
+        user.setLogin("Misha4433");
+        user.setPassword("lutsk12345");
+        user.setAge(41);
+        registrationService.register(user);
+        assertTrue("Misha4433".equals(user.getLogin()));
+    }
+
+    @Test
+    void userLoginIsAlreadyExist_NotOk() {
+        User user = new User();
+        user.setLogin("Vitaliks");
+        user.setAge(62);
+        user.setPassword("great_password");
+        Exception exception = assertThrows(CustomeException.class, () ->
+                registrationService.register(user));
+        String excepted = "User with this login is already register!";
+        String actual = exception.getMessage();
+        assertTrue(actual.contains(excepted));
+    }
+
+    @Test
+    void userPasswordIsNotExist_NotOk() {
+        User user = new User();
+        user.setLogin("Hensai");
+        user.setAge(19);
+        user.setPassword("");
+        assertThrows(CustomeException.class, () -> registrationService.register(user));
+    }
+
+    @Test
+    void userPasswordHad3letters_NotOk() {
+        User user = new User();
+        user.setLogin("Igorenis");
+        user.setAge(42);
+        user.setPassword("kgr");
+        Exception exception = assertThrows(CustomeException.class, () ->
+                registrationService.register(user));
+        String expected = "User password is need to be at least 6 letters!";
```

Comment:
The expected message 'User password is need to be at least 6 letters!' is grammatically incorrect. Ensure the expected message matches the corrected exception message.